packages/api-page-builder-import-export/__tests__/pageImportExportTask/useHandler.ts

```diff
@@ -21,11 +21,16 @@ export default (params: Params = {}) => {
         {
             type: "context",
             apply: (context: PbContext) => {
-                if (context.i18nContent) {
+                if (context.i18n) {
                     return;
                 }
-                context.i18nContent = {
-                    ...(context.i18nContent || ({} as any)),
+
+                context.i18n = {
+                    // @ts-ignore
```

Comment:
I think a little more reasoning around why this is tsignored would be helpful to others

Reply:
Try removing the ignore 😂  Joking aside, this whole file needs proper restructuring, and we can open a dedicated cleanup issue for that. I'll open an issue right after we merge this PR, and will take care of it in the near future.